src/doc/man/cargo-doc.md

```diff
@@ -31,6 +31,15 @@ option.
 
 {{#option "`--no-deps`" }}
 Do not build documentation for dependencies.
+
+**Note:** Documentation is _cumulative_. If you’ve already run
```

Comment:
The cumulative behavior is not specific to the `--no-deps` flag. I would personally prefer to seeing this in DESCRIPTION than here.